src/App.js

```diff
@@ -1,26 +1,24 @@
-import React from 'react';
```

Comment:
I did not actually change anything in this file.  I think I just looked around in here and then saved, and that automatically triggered a formatting change that is showing up here.  Sorry!